gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_3_0/schema.yml

```diff
@@ -13,7 +13,7 @@ databaseChangeLog:
               - column: {name: reference_type, type: nvarchar(64), constraints: { nullable: false } }
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.